src/cascadia/TerminalCore/TerminalSelection.cpp

```diff
@@ -342,6 +342,17 @@ void Terminal::UpdateSelection(SelectionDirection direction, SelectionExpansion
         break;
     }
 
+    // 2.B) Clamp the movement to the mutable viewport
+    const auto mutableViewport{ _GetMutableViewport() };
+    if (targetPos > mutableViewport.BottomRightInclusive())
+    {
+        targetPos = mutableViewport.BottomRightInclusive();
+    }
+    else if (targetPos < mutableViewport.Origin())

```

Comment:
i bet backporting to 1.14 will require manual intervention because you're using the `til::point` comparison operator...

Reply:
ughhh. I'll make a port rn.